brainscore/submission/evaluation.py

```diff
@@ -70,7 +70,8 @@ def run_evaluation(config_dir, work_dir, jenkins_id, db_secret, models=None,
                 for model_name in test_models:
                     model_entry, created = Model.get_or_create(name=model_name, owner=submission_entry.submitter,
                                                                defaults={'public': submission_config.public,
-                                                                         'submission': submission_entry})
+                                                                         'submission': submission_entry,
+                                                                         'competition': submission_config.competition_submission})
```

Comment:
if I understand the current flow right, this will write a bool rather than `"cosyne2022"`. I think we should write the name of the competition though